docker/dev.Dockerfile

```diff
@@ -1,20 +1,20 @@
-FROM python:3.12-bookworm
+FROM ghcr.io/astral-sh/uv:python3.12-bookworm-slim
 ENV TZ=UTC
 
 WORKDIR /app
 
-RUN apt update -y
-RUN apt install \
+RUN apt-get update && apt-get install -y \
     build-essential \
     curl \
     npm \
-    -y
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
 COPY . /app
 
-RUN pip install poetry
-RUN poetry config virtualenvs.create false
-RUN poetry install --no-interaction --no-ansi
+# Install dependencies using uv
+RUN --mount=type=cache,target=/root/.cache/uv \
+    uv pip install --system -r requirements.txt
```

Comment:
requirements.txt?

Reply:
I think we actually want 

RUN --mount=type=cache,target=/root/.cache/uv \
    uv sync --python-preference system
    
because unlike pip it will look at the uv.lock files. maybe we want --frozen?

I think `--python-preference only-system` which like pip's `--system` will use the system python install instead of a uv managed python. If it doesn't match the requirements in the toml it will fail.

could also do:

    ENV UV_PYTHON_PREFERENCE=only-managed